csharp/test/Ice/compress/Test.ice

```diff
@@ -18,9 +18,9 @@ exception MyException
 
 interface TestIntf
 {
-    [compress:params] void opCompressParams(int size, ByteSeq p1);
-    [compress:return] ByteSeq opCompressReturn(int size);
-    [compress:params,return] ByteSeq opCompressParamsAndReturn(ByteSeq p1);
+    [compress(params)] void opCompressParams(int size, ByteSeq p1);
+    [compress(return)] ByteSeq opCompressReturn(int size);
+    [compress(params,return)] ByteSeq opCompressParamsAndReturn(ByteSeq p1);
```

Comment:
I think it should be formatted like a function call:
```
[compress(params, return)]
```

Reply:
Fixed! I also checked everywhere else we used commas in metadata, and this was the only place we did (apart from stuff like `cpp:type(map<int, int>)`)